scripts/updateUsersFromLogs.js

```
@@ -0,0 +1,185 @@
+      users.push(file);
+    }
+
+    console.log(`Found ${users.length} user directories`);
+
+    // Define batch size for processing
+    const batchSize = 5;
+
+    // Process users in batches
+    for (let i = 0; i < users.length; i += batchSize) {
+      const batch = users.slice(i, i + batchSize);
+      await Promise.all(batch.map(user => updateUserAndWorkers(user)));
```

ESLint

[error] Replace `user` with `(user)`

(prettier/prettier)